package.json

```diff
@@ -5,7 +5,8 @@
   "author": "Michael Davis",
   "bugs": "https://github.com/Contentstack-Solutions/contentstack-cli-content-type/issues",
   "dependencies": {
-    "@contentstack/cli-command": "0.1.0-beta",
+    "@contentstack/cli-command": "^1.2.8",
+    "@contentstack/cli-utilities": "^1.4.4",
     "@oclif/command": "^1.8.0",
     "@oclif/config": "^1.17.0",
```

Comment:
These two packages are no more needed, right? As we should use `@oclif/core`

Reply:
@antonyagustine removed